modules/deleting-an-accelerator-profile.adoc

```diff
@@ -0,0 +1,34 @@
+:_module-type: PROCEDURE
+
+[id="deleting-an-accelerator-profile_{context}"]
+= Deleting an accelerator profile
+
+[role='_abstract']
+To discard accelerator profiles that you no longer require, you can delete them so that they do not appear on the dashboard.
+
+.Prerequisites
+* You have logged in to {openshift-platform}.
+* You have the `cluster-admin` role in {openshift-platform}.
+* The `acceleratorprofiles.dashboard.opendatahub.io` custom resource definition (CRD) exists in your deployment and contains an accelerator profile.
```

Comment:
You've updated 'contained' to 'has' elsewhere in this PR, if that's a 'plain language' swap we want to make consistently everywhere.